README.md

```diff
@@ -1,5 +1,7 @@
 # CNCF Special Interest Group for Security (SIG-Security)
 
+<img src="https://github.com/cncf/artwork/blob/master/other/sig-security/horizontal/color/sig-security-horizontal-color.png" width="300">
```

Comment:
Please use the SVG!!!

https://www.cncf.io/blog/2019/07/17/what-image-formats-should-you-be-using-in-2019/